fluentui_controls/src/main/java/com/microsoft/fluentui/tokenized/controls/TextField.kt

```diff
@@ -0,0 +1,217 @@
+                            .onFocusChanged { state ->
+                                isFocused = false
+                                when {
+                                    state.isFocused ->
+                                        isFocused = true
+                                }
+                            },
+                        singleLine = true,
+                        keyboardOptions = keyboardOptions,
+                        keyboardActions = keyboardActions,
+                        visualTransformation = visualTransformation,
+                        decorationBox = @Composable { innerTextField ->
+                            if (value.isEmpty() && !hintText.isNullOrBlank()) {
+                                Box(
+                                    Modifier.fillMaxWidth(),
+                                    contentAlignment = if (LocalLayoutDirection.current == LayoutDirection.Rtl)
+                                        Alignment.CenterEnd
+                                    else
+                                        Alignment.CenterStart
+                                ) {
+                                    BasicText(
+                                        hintText,
+                                        style = textFieldTokens.hintTextTypography(textFieldInfo).merge(
+                                            TextStyle(
+                                                color = textFieldTokens.hintColor(textFieldInfo)
+                                            )
+                                        )
+                                    )
+                                }
+                            }
+                            innerTextField()
+                        },
+                        textStyle = textFieldTokens.inputTextTypography(textFieldInfo).merge(
+                            TextStyle(
+                                color = textFieldTokens.inputTextColor(textFieldInfo),
+                                textDirection = TextDirection.ContentOrLtr
+                            )
+                        ),
+                        cursorBrush = textFieldTokens.cursorColor(textFieldInfo)
+                    )
+                    if (!rightAccessoryText.isNullOrBlank()) {
+                        Spacer(Modifier.requiredWidth(8.dp))
+                        BasicText(
+                            rightAccessoryText,
+                            modifier = Modifier.padding(vertical = 12.dp),
+                            style = textFieldTokens.rightAccessoryTextTypography(textFieldInfo).merge(
+                                TextStyle(
+                                    color = textFieldTokens.rightAccessoryTextColor(textFieldInfo)
+                                )
+                            )
+                        )
+                    }
+                    if (value.isNotBlank() && rightAccessoryIcon?.isIconAvailable() == true) {
```

Comment:
remove == check as it is implicit check

Reply:
Since rightAccessoryIcon is nullable, this check is required and suggested by AndroidStudio. If rightAccessoryIcon comes as null, the output of rightAccessoryIcon?.isIconAvailable() will be null